packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -429,6 +440,10 @@ export function usePress(props: PressHookProps): PressResult {
       };
 
       pressProps.onMouseUp = (e) => {
+        if (!e.currentTarget.contains(e.target as HTMLElement)) {
+          return;
+        }
+
         if (!state.ignoreEmulatedMouseEvents && e.button === 0) {
           triggerPressUp(e, state.pointerType);
         }
```

Comment:
Make sure we get all of the events assigned to pressProps. I think maybe touch events are missing? Is there a reason they were not included?

Reply:
i think i had placed them all the functions with an event, but i couldn't tell if they were doing anything or not. should i go ahead and place them in all the pressProps events?

Reply:
Yeah all of the ones that are on pressProps, not global listeners.